src/components/app/DocumentList.js

```diff
@@ -186,7 +186,9 @@ class DocumentList extends Component {
             this.setState({
                 cachedSelection: null
             }, () => {
-                dispatch(setListIncludedView());
+                if (includedView.windowType === 'pickingSlot') {
```

Comment:
@wiadev 

about 3. I am a bit concerned about that. While the fix is solving our current problem (which is in Picking modal popup) it's not solving it generally. And that's also important for us because these features are basically our building blocks for more and more functionalities (like LEGO :) ).

I think we shall create a new task to refactor and do it right. On that task we shall also discuss about it, i can also give you more input. We don't have any pressure on that. The "high prio" of this task was exactly because we need to deliver the Picking window ASAP.
wdyt?